src/modules/editor/EditorManager.tsx

```diff
@@ -65,6 +67,37 @@ export class EditorManager {
     width: 320,
     height: 180
   };
+
+  private envData: EnvData = {
```

Comment:
We should add a `FIXME` as the backend would provide the example code when creating a project, rather than the frontend providing it